cfme/v2v/conversion_hosts.py

```diff
@@ -0,0 +1,237 @@
+               cluster,
+               hostname,
+               conv_host_key,
+               transformation_method,
+               vddk_library_path=None,
+               vmware_ssh_key=None,
+               osp_cert_switch=None,
+               osp_ca_cert=None):
+        """Configure new conversions host in UI
+        Args:
+            target_provider: RHV or OSP
+            provider_name: rhv43 or osp13-ims
+            cluster: Default
+            hostname: Hostname to be configured
+            conv_host_key: /etc/pki/ovirt-engine/keys/engine_id_rsa on RHV-M.
+            transformation_method: vddk or ssh
+            vddk_library_path: url for vddk
+            vmware_ssh_key: /var/lib/vdsm/.ssh/id_rsa on rhv host
+            osp_cert_switch: Yes or No
+            osp_ca_cert: Tls Cert
+
+        """
+        view = navigate_to(self, "Configure")
+        provider_type = ("Red Hat Virtualization" if target_provider.one_of(RHEVMProvider)
+                         else "Red Hat OpenStack Platform")
+        view.location.fill({"provider_type": provider_type,
+                            "provider_name": provider_name,
+                            "cluster": cluster})
+        view.hosts.wait_displayed()
+        view.hosts.fill({"hostname": hostname})
+        view.authentication.wait_displayed()
+        view.authentication.fill({"conv_host_key": conv_host_key,
+                                  "transformation_method": transformation_method,
+                                  "vddk_library_path": vddk_library_path,
+                                  "vmware_ssh_key": vmware_ssh_key,
+                                  "osp_cert_switch": osp_cert_switch,
+                                  "osp_ca_cert": osp_ca_cert})
+        view.close_btn.is_displayed
+        view.close_btn.click()
+        return self.instantiate(target_provider=target_provider,
+                                provider_name=provider_name,
+                                cluster=cluster,
+                                hostname=hostname,
+                                conv_host_key=conv_host_key,
+                                transformation_method=transformation_method,
+                                vddk_library_path=vddk_library_path,
+                                vmware_ssh_key=vmware_ssh_key,
+                                osp_cert_switch=osp_cert_switch,
+                                osp_ca_cert=osp_ca_cert
+                                )
+
+    def in_progress(self, hostname):
+        """Wait for conversion host to configure"""
+        view = navigate_to(self, "All")
+        try:
+            wait_for(
+                lambda: not view.conversion_host_progress.in_progress(hostname),
+                message="Conversion Host configuration is in progress",
+                delay=5,
+                num_sec=300
+            )
+        except TimedOutError:
+            self.logger.warning("Timed out waiting for configuring...")
```

Comment:
Required: some detail about what you're configuring, e.g. the hostname would be useful.